app/models/user.rb

```diff
@@ -10,10 +10,18 @@ class User < ApplicationRecord
 
   has_one_attached :profile_photo
 
+  def name
+    first_name.to_s + ' ' + last_name.to_s
```

Comment:
why name.to_s??

Reply:
if name is nil then it gives error that undefined method + for NilClass

Reply:
Lets use interpolation here then you dont have to do to_s